client/package.json

```diff
@@ -8,13 +8,16 @@
     "bootstrap": "^4.6.0",
     "bootswatch": "^4.6.0",
     "fortmatic": "^2.2.1",
+    "moment": "^2.29.1",
     "react": "16.11.0",
     "react-bootstrap": "^1.4.3",
     "react-dom": "16.11.0",
+    "react-icons": "^4.1.0",
     "react-particles-js": "^3.4.1",
     "react-router-bootstrap": "^0.25.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.2.0",
+    "react-share": "^4.3.1",
```

Comment:
You'll need to re-add these with yarn to save them into `client/yarn.lock`. Also don't think `moment` is being used yet. See the bottom of the README: https://github.com/janclarin/blossym#adding-new-dependencies (all commands run from the root dir).

Reply:
Oops, nvm, there's only the single yarn.lock file now.